### DISCUSSION TITLE
Only 1 peer but 0 download speed

### DISCUSSION THREAD

**Discussion by thisplantguy** [transmission/transmission#8444] [Support]
### Discussed in https://github.com/transmission/transmission/discussions/4691

<div type='discussions-op-text'>

<sup>Originally posted by **thisplantguy** January 29, 2023</sup>
I'm running the latest Transmission Beta for Apple Silicon, and have come across this phenomenon a few times now. Basically, if there's only 1 other peer, oftentimes I simply don't get any data downloaded.  I'd rather not share what torrent this is, but I'm attaching a screenshot of what it looks like. I'm unsure of why this happens frequently, but it seems odd that I'd have a peer but no information exchanged. 
![Screenshot 2023-01-29 at 7 34 45 PM](https://user-images.githubusercontent.com/51683498/215382554-538b0de3-472b-40bf-9da0-7eab3ff19caa.png)

Beyond hoping/waiting for more peers, is there anything I can do about this? I should mention that occasionally it will show more than 1 peer (2, 3, even 4) and the speed (or lack theirof) stays the same.</div>

**Comment by Pentaphon** [comment]
It's fairly common to connect to 1 peer and never get any data for a while or even through the life of the torrent. There's any number of reasons why it doesn't fully connect, but I've seen this behavior on most torrent clients. It happens 5% of the time on single peer public torrents, I'd say, and I've seen many people online say they have 1 peer but no data coming from it. 1 peer is not enough to make a "swarm" and not enough to reliably transfer data. Have you tried this with other clients? Have you asked other people to connect to the same torrent in case they connect better to the single peer than you do, which is the usual workaround for data not moving. What client is the peer using? Have you made sure to open a port for your connection? Do you have any kind of unusual setup that could make the problem worse?

**Comment by ckerr** [comment]
https://github.com/transmission/transmission/pull/5176 may help with this one. @thisplantguy please check out 4.0.2 when it comes out next week and LMK if things improve or not :bow:
